lib/travis/addons/config/notify.rb

```diff
@@ -89,6 +89,11 @@ def previous_build_failed?
             !previous_build_passed?
           end
 
+          def all_jobs_finished?
+            finish_states = %i[passed failed errored canceled].freeze
```

Comment:
@piotr-travisci, I would also make it a constant

Reply:
Thank you @pavel-d 
I refactored it to constant and added test case for it.